### ISSUE TITLE
Significant digits in Ref Points table

### ISSUE THREAD

**Issue by taylori** [melmonk/StockAssessment_template#13]
The scale of the values in the Reference Points table (Table f) is very broad, making the choice of digits a bit of a pain. Based on discussion with John Field and Andi Stephens I think it would be useful to have new rules for significant digits in this table and/or easier user controls for changing the existing rules (based on `ifelse` statements starting around line 517 of Exec_summary_figs_tables.R (https://github.com/melmonk/StockAssessment_template/blob/master/Rcode/Exec_summary_figs_tables.R#L517), the default is 1 digit after decimal for values greater than 1.0 and 4 digits for values below 1.0, but this could be refined.

Also, it might be useful to use `format` to print full digits. In the Yellowtail Rockfish assessment, the Unfished spawning output was 14.996 trillion eggs. The status quo setup rounded this to 15.0 which then was printed as 15, when 15.0 would have been better.
```
> as.character(15.0)
[1] "15"
> format(15.0, nsmall=1)
[1] "15.0"

```

**Comment by CWetzel** [comment]
I want to chime in here as well.  I used this command to augment the
decimal points:

print.numeric<-function(x, digits) { formatC(x, digits = digits, format =
"f") }

in this year's POP ES section tables and I think it worked well.  I agree
that we should fix up the Reference Points table.  However, I tried to have
a mix of decimal points in this table and was unsuccessful, but don't
remember exactly what the issue was. Perhaps Melissa has a better memory
for why it was set up the way it was.

Another related issue to the decimal point in the table was adding commas
to values in the text.  I used this clunky function in the ES section, but
perhaps there is a better way:

`r paste(prettyNum(Spawn_mod1,big.mark=","), fecund_unit,sep=' ')`

I think as a group (just to clarify not just Melissa) we should discuss how
best to improve on these issues.

**Comment by melmonk** [comment]
I agree on all the points you've both made.  I was going to look at the range of number styles used in the assessment tables from this past cycle and modify the template based on that.  In xtable, multiple number formats/styles within one table is a pain in R markdown.

I recently learned of the [Huxtable package](https://hughjonesd.github.io/huxtable/) and looks like that might help solve this issue.  I haven't gotten around to exploring it yet.

Know that this is all on my radar.  It would be good to get a group together who used the template and discuss improvements, i.e., I know Chantel and I created child documents for each section to help with organizing sections and also debugging.
